src/corelibs/U2Script/src/UgeneContextWrapper.cpp

```diff
@@ -86,6 +86,8 @@
 #include <U2Lang/QueryDesignerRegistry.h>
 #include <U2Lang/WorkflowEnvImpl.h>
 
+#include <BundleInfoMac.h>
```

Comment:
nit: technically it is not bundle for "mac" only anymore and can be used for any platform. This will remove all ifdefs from the code outside of the BundleInfo.